### PR TITLE
LPD-23332 Specify opaque colors only for disabled elements

### PR DESCRIPTION
**Pull request by igor-franca**
Related tickets: 
[LPP-53625](https://liferay.atlassian.net/browse/LPP-53625)
[LPP-50121](https://liferay.atlassian.net/browse/LPP-50121)
[LPP-50888](https://liferay.atlassian.net/browse/LPP-50888)

The main objective of this PR is to specify low opacity states within the form fields.

### CODE REVIEW

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -100,29 +100,16 @@ body .ddm-user-view-content {
 .lfr-ddm-form-container {
 	word-break: normal;
 
-	.btn {
-		opacity: unset;
-	}
+	.custom-control-input:disabled ~ .custom-control-label {
+		::before {
+			background-color: #0b5fff;
```
Comment (cgmonte) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:105]: Hey @igor-franca, as we talked before, you should use `$primary` variable here, as per [Clay UI Color](https://clayui.com/docs/css/color.html).

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -100,29 +100,16 @@ body .ddm-user-view-content {
...
+			border-color: #0043c5;
```
Comment (cgmonte) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:106]: This is not a standard Clay color, but I think this close to `$primary-d2`, please check with Julia.

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 3]
```diff
@@ -100,29 +100,16 @@ body .ddm-user-view-content {
...
+		}
 
-	.custom-control-label-text {
 		color: #6b6c7e;
```
Comment (cgmonte) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:109]: This here is `$secondary`.

**Comment by cgmonte** [comment]
Hey @igor-franca, I could not reproduce the issues in my local machine. I think it would be better to see it in your machine, if possible. I will reach to you tomorrow.

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by igor-franca** [comment]
ci:test:stable

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**